.storybook/stories.js

```diff
@@ -25,12 +25,19 @@ export const onThemeSwitch = context => {
   };
 };
 
+const funcSnippet = (selectedValue) => {
+  console.log("funcSnippet", selectedValue)
```

Comment:
if this is temporary code only for debugging let's remove it before megring

Reply:
Done